### ISSUE TITLE
MVP delivery

### ISSUE THREAD

**Issue by barto74** [dropapp/dropapp#50]
We need to make a checklist of all the things that needs to be done to hand over our prototype of this week to the IHA people.

- [x] Define how they get support
- [x] Invite them to Slack
- [x] How would the hosting / domain look like during the development process?
- [x] How do we update in git?
- [x] Discuss the option of a second check up visit with paid flights and accomodation.
- [x] Usage of barcode labels (buy them [here](https://officemarket.gr/product/50351/))

Please elaborate on the list itself and on the issues on the list

**Comment by HaGuesto** [comment]
Discuss the option of a second check up visit? After a month or so?

**Comment by barto74** [comment]
1. Delivery notes being written
2. Support in #support channel on our Slack
3. Hosting temporary on drop-app.de subdomain until we close #26 
4. Git: we have started working with branches and pull requests, no more directly push to master
5. We started the discussion. @HaGuesto you are in touch about it?!
6. Barcodes are out of MVP.

Everybody agree on closing the issue or is there anything to add? I'd rather make separate tasks for the things that still need to be done @HaGuesto @bfirsh
